NOTES.md

```diff
@@ -0,0 +1,3 @@
+`ember-cli-moment-shim` is a depedency of this addon since it is used by the addon code and since we want to provide this package so that is can be used by a consumer.
```

Comment:
Do we need this explanation?  Isn't this the same explanation that would apply to all dependencies?  What makes this one different?

If were keep it `is` should be changed to `it`.